### PR TITLE
[GEN-694] Remove fusions from dashboarding

### PR DESCRIPTION
**Pull request by thomasyu888**
* Remove fusions from dashboarding
* Remove models

### CODE REVIEW

File: genie/dashboard_table_updater.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def update_cumulative_sample_table(
     seg_counts["Total"] = sum(seg_counts)
 
     total_counts = pd.DataFrame(clinical_counts)
-    total_counts["Fusions"] = fusion_counts
+    total_counts["Fusions"] = sv_counts
```
Comment (rxu17) [inline genie/dashboard_table_updater.py:152]: Any reason we don't update the `total_counts["Fusions"]` to `total_counts["Structural_Variants"]` or something like that? Similar question for Lines 266, 459, 465?

**Review by rxu17 — APPROVED** [review]
LGTM (after the linting resolution) ! Just a question

File: genie/dashboard_table_updater.py (hunk 1 above)
Comment (thomasyu888) [reply genie/dashboard_table_updater.py:152]: @rxu17 That is a great question.  We should create a ticket to rename "Fusions" to "Structural_Variants", BUT... All fusions are structural variations and vice versa.

Comment (rxu17) [reply genie/dashboard_table_updater.py:152]: Makes sense!
I just wasn't sure if it mattered that it affected the appearance of the `cumulative_sample_table` in synapse in that it still used `Fusions` as the naming convention

Comment (thomasyu888) [reply genie/dashboard_table_updater.py:152]: Ya... the tables themselves still use `Fusions` as the column header so thats why. We would need to update the headers in the Synapse tables along with the code which is why those are the same.

**Pull request merged**